### ISSUE TITLE
not following symlink for package options?

### ISSUE THREAD

**Issue by rwv37** [jrmarino/synth#228]
I recently have been reorganizing my machine's configuration (in general, not just for synth, nor just for ports/packages) by using symlinks. So for example ``/var/db/ports/ports-mgmt_synth`` is actually a symlink to ``/stuff/stowfiles/synth/var/db/ports/ports-mgmt_synth``. I *thought* everything had been going well for a while now, but I think I just discovered something:

I have one and only one port defined to have an options file within ``/var/db/ports/``: synth itself. The reason I have this is to disable the watchdog timer, which in my experience is too short to allow synth (on my machine) to build certain ports. For example, with the watchdog timer enabled, synth will typically kill the build of ``lang/dotnet`` after chugging along at it for like an hour and a half, maybe two hours. Disabling the watchdog timer enabled synth to build this port (and others in a similar situation) just fine.

However, I recently had to build ``lang/dotnet`` for the first time in a while, and the watchdog timer is killing it prematurely again. Through experimentation, it seems to me that synth pays attention to ``/var/db/ports/ports-mgmt/synth/options`` if and only if I have that as a "real" directory and file, not a symlink.

Is this as expected? And regardless of expected or not, is it as it should be?

For what it's worth, ``make config`` from within ``/usr/ports/ports-mgmt/synth`` _does_ successfully read and modify the linked file (i.e. ``/stuff/stowfiles/synth/var/db/ports/ports-mgmt_synth``).

Thanks.

**Comment by jrmarino** [comment]
Yes, I would say that's expected.  In particular synth (unlike it's descendent ravenadm) uses the native file packages, and those don't even recognize symlinks (those packages are generic and work for other platforms like windows).  So I would bet that there's a basic type check for a directory and a symlink would fail that check.
